### DISCUSSION TITLE
selenide open command doesnot maximize the browser window

### DISCUSSION THREAD

**Discussion by aravindraveendran7** [selenide/selenide#2373] [Q&A] [answered]
I tried the command open("https://www.google.com"), and even though the browser is launched and opening the URL, the same is not getting maximized. Please advise.

**Comment by aravindraveendran7** [comment, marked as answer]
Found the solution by using, WebDriverRunner.getWebDriver().manage().window().maximize()

**Reply by asolntsev** [reply]
@aravindraveendran7 Yes, it works, BUT I think it's rather a bad practice. Read explanation here: https://github.com/selenide/selenide/issues/810

**Reply by aravindraveendran7** [reply]
Make sense. Thank you @asolntsev .
